testnet/shared-config-test.json

```diff
@@ -500,6 +500,78 @@
           "decimals": 18
         }
       ]
+    },
+    {
+      "id": 12,
+      "chainId": 3799,
+      "parachainId": 1000,
```

Comment:
Is this correct parachainId?

Reply:
they are a standalone chain and dont have a parachainID, this is just a dummy one, as long as its different from other substrate chain, it should be good

Reply:
This could be problematic in SDK as I believe this parachain ID is used to transform addresses. Can we maybe leave it empty?